app/views/directives/route-service-bar-chart.html

```diff
@@ -0,0 +1,15 @@
+<div class="route-service-bar-chart">
+  <h5>Traffic Split</h5>
+  <div ng-repeat="backend in routeServices.backends">
```

Comment:
No UID available, we just have name. We could do a `track by backend.name`, but I think it's possible for the same name to be there twice (even if it doesn't make sense), which would cause a runtime error.